app/assets/javascripts/spotlight/multi_image_selector.js

```diff
@@ -1,7 +1,7 @@
 // Module to add multi-image selector to widget panels
 
 (function(){
-  $.fn.multiImageSelector = function(image_versions, clickCallback) {
+  $.fn.multiImageSelector = function(image_versions, clickCallback, activeTilesource) {
```

Comment:
Not sure if this param should come after the callback.  We don't use this necessarily in the other multi-image contexts.